docs/api-ref/prefect/projects/base.md

```diff
@@ -1,10 +1,10 @@
 ---
-description: Prefect Python API for projects.
```

Comment:
Do we want to rename the `projects` directory to `deployments`?

Reply:
👍